src/azure-cli/azure/cli/command_modules/util/_help.py

```diff
@@ -0,0 +1,39 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+# pylint: disable=line-too-long, too-many-lines
+
+helps['rest'] = """
+type: command
+short-summary: Invoke a custom request.
+long-summary: >
+    This command automatically authenticates using the logged-in credential: If Authorization header is not set, it
```

Comment:
Changed the help according to https://github.com/Azure/azure-cli/pull/13249#discussion_r423436138